_faqs/03-28-which-version-of-opensearch-should-i-use.markdown

```diff
@@ -0,0 +1,10 @@
+---
+question: Which version of OpenSearch should I use?
+category: Upgrading to OpenSearch
+---
+
+or OpenSearch and other software in the OpenSearch project, new features and active development always takes place against the newest version. The OpenSearch project follows the [semantic versioning specification](https://opensearch.org/blog/technical-post/2021/08/what-is-semver/) for assigning version numbers to releases, so in most cases you should be able to upgrade to the latest version of any software without encountering incompatible changes.
```

Comment:
I think there is a missing "F" at the start of this paragraph

Reply:
🤦‍♂️ Fixed, thanks.